Content/Coronavirus’ Impact on Business/Index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Coronavirus’ Impact on Business - Save Yours!
+date: 2020-09-09T16:00:00.284Z
+description: How to save your business during Coronavirus pandemic
+author: Abeer Albada
+author_company: Zaat.dev
+author_twitter: AbeerAlbada
+author_github: abeeralbada
+image: ./Coronavirus_Impact_on_Business.png
+tags: Business, Coronavirus, Web Development, Software Solutions
+---
```

Comment:
You are missing `issue-number` here
Make a new issue from here, https://github.com/zaatdev/blog/issues